benchmarks/ecal_gaps/org2py.awk

```diff
@@ -0,0 +1,22 @@
+BEGIN {
```

Comment:
Can this be centralized instead of copied in each directory? It's also in benchmarks/background.